mythril/laser/ethereum/state.py

```diff
@@ -156,7 +208,7 @@ def memory_write(self, offset, data):
     def pop(self, amount=1):
         """ Pops amount elements from the stack"""
         if amount >= len(self.stack):
-            raise IndexError()
+            raise StackUnderflowException
```

Comment:
```StackUnderflowException()```?

Reply:
there is no need for an argument to be passed to the `StackUnderflowException` right? so i don't think it will be necessary here.